parser/model/model_test.go

```diff
@@ -274,6 +274,101 @@ func TestJobCodec(t *testing.T) {
 	require.NoError(t, err)
 	require.True(t, isDependent)
 
+	// Test IsDependentOn for exchange partition with table.
+	// test ActionCreateSchema and ActionExchangeTablePartition is dependent.
```

Comment:
I think ActionDropSchema and ActionDropTable are also interesting to test for completeness.

Reply:
thanks, it has been added.